eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -28,15 +28,11 @@ stages:
                 deploy:
                   steps:
                     - checkout: self
-                    - pwsh: |
-                        Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}
-                        New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
-                        $underscorePrefix = "${{artifact.name}}"
-                        $dashPrefix = "${{artifact.name}}".Replace("_", "-")
-                        Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$dashPrefix-[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
-                        Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$underscorePrefix-[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
-                        Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
-                      displayName: Stage artifacts
+                    - template: /eng/pipelines/templates/steps/stage-filtered-artifacts.yml
```

Comment:
Thanks for the template there appear to be other instances we could use this in this pipeline. 

BTW it is fine to do this as a separate clean-up issue.